packages/client/lib/rpc/modules/engine.ts

```diff
@@ -467,6 +469,48 @@ export class Engine {
       return response
     }
 
+    if (block._common.isActivatedEIP(4844)) {
+      let validationError: string | null = null
+      if (versionedHashes === undefined) {
+        validationError = `Error verifying versionedHashes: received none`
+      } else {
+        // Collect versioned hashes to match
+        const txVersionedHashes = []
+        for (const tx of block.transactions) {
+          if (tx instanceof BlobEIP4844Transaction) {
+            for (const vHash of tx.versionedHashes) {
```

Comment:
maybe rename to `vHashes`?

Reply:
but this is individual hash being pushed to create a flat txVersionedHashes

Reply:
Ah sorry, misread the code